TestAssets/TestProjects/MSBuildTestApp/MSBuildTestApp.csproj

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I'd suggest not focusing too much on the contents of this file in this PR. @davkean is busily cleaning up required properties and building a new template. This should a) work now and b) over time be kept roughly in line with the current best practice.